tools/webidl_binder.py

```diff
@@ -306,8 +306,13 @@ def base_type_to_c(t):
     return ret
 
   t = t.replace(' (Wrapper)', '')
```

Comment:
i think maybe all the lines from here to the end could be written with less repetition, like this:
```
prefix = ''
suffix = ''
if '[]' in t:
  t = t.replace('[]', '')
  suffix = '*'
if 'const ' in t:
  t = t.replace('const ', '')
  prefix = 'const '
return prefix + base_type_to_c(t) + suffix
```
What do you think?

Reply:
I agree, that looks a lot cleaner.